packages/react/src/runtimes/external-store/ThreadMessageLike.tsx

```
@@ -100,19 +99,19 @@ export const fromThreadMessageLike = (
 
               case "tool-call": {
-                if ("argsText" in part) {
-                  if ("args" in part) {
-                    return part;
-                  }
-
+                if (part.args) {
                   return {
                     ...part,
-                    args: parsePartialJson(part.argsText),
+                    toolCallId: part.toolCallId ?? "tool-" + generateId(),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)